### PR TITLE
More middlewares

### PR DESCRIPTION
**Pull request by eirikurn**
Refactored tux data fetching into universal router and tux middleware.
Added helmet (+ middleware) for SSR head management.

Now SSR is pretty good except it's missing CSS. Neutrino currently doesn't extract css into a file. Will fix soon.

### CODE REVIEW

File: packages/tux-example-site/src/routes/Home.js [hunk 1]
```diff
@@ -36,7 +34,7 @@ const Home = ({ pages, articles, sellPoints, pricetable, testimonial, gallery, c
         <Testimonial testimonial={testimonial.items}/>
       </Section>
       <Section backgroundColor="#F5F7FA">
-        <H1>What People Say About Tux</H1>
+        <H1>BLABLA People Say About Tux</H1>
```
Comment (rthor) [inline packages/tux-example-site/src/routes/Home.js]: lol

File: packages/tux-example-site/src/server.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import 'babel-polyfill'
```
Comment (rthor) [inline packages/tux-example-site/src/server.js:1]: Is this needed on the server?

**Review by rthor — APPROVED** [review]

File: packages/tux-example-site/src/routes/Home.js (hunk 1 above)
Comment (eirikurn) [reply packages/tux-example-site/src/routes/Home.js]: Hehe, whoops

File: packages/tux-example-site/src/server.js (hunk 2 above)
Comment (eirikurn) [reply packages/tux-example-site/src/server.js:1]: Yes, for regeneratorRuntime. Though I noticed that neutrino-preset-web adds it magically in #66. I'll do the same on the server.

**Pull request merged**